openedx/features/enterprise_support/api.py

```diff
@@ -268,7 +293,7 @@ def consent_needed_for_course(user, course_id):
     return consent_necessary_for_course(user, course_id)
 
 
-def get_enterprise_consent_url(request, course_id, user=None, return_to=None):
+def get_enterprise_consent_url(request, course_id, user=None, return_to=None, return_to_url=None):
```

Comment:
@haikuginger Yup, agreed that there is no way to get around having to distinguish between cases where `course_id` needs to be used to build the redirect URL. But I think that with the change that you mention, we could make the logic easier to follow. Or at the very least it'd make the signature of the function less surprising. So I'm in favor of making that change.